Bio/PDB/parse_pdb_header.py

```diff
@@ -144,7 +190,9 @@ def _parse_pdb_header_list(header):
         'structure_reference': "unknown",
         'journal_reference': "unknown",
         'author': "",
-        'compound': {'1': {'misc': ''}}, 'source': {'1': {'misc': ''}}}
+        'compound': {'1': {'misc': ''}}, 'source': {'1': {'misc': ''}},
+        'has_missing_residues': False,
+        'missing_residues' : []}
```

Comment:
You'll need to remove a space here for the automated PEP8 style checks which are failing on TravisCI,

```
$ flake8 Bio/
Bio/PDB/parse_pdb_header.py:195:27: E203 whitespace before ':'
```

See the ``CONTRIBUTING.rst`` file for more details if you want to run this check locally.